src/features/defi/components/GetStarted/GetStarted.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, VStack } from '@chakra-ui/react'
+import osmosis from 'assets/osmosis.svg'
+import { Text } from 'components/Text'
+
+import { DefiModalHeader } from '../DefiModalHeader/DefiModalHeader'
+
+type GetStartedProps = {
+  assetId: string
+}
+
+const CAIP19_TO_MAX_APR = {
+  'cosmoshub-3/slip44:118': '12'
+}
+
+export const GetStarted = ({ assetId: caip19 }: GetStartedProps) => {
+  // TODO: parentheses are nice but let's get asset name from selectAssetByCAIP19 instead of this
+  const asset = (_ => ({
+    name: 'Osmo'
+  }))(caip19)
+  const maxApr = CAIP19_TO_MAX_APR['cosmoshub-3/slip44:118']
+  return (
+    <Box pt='51px' pb='20px' px='24px'>
+      <Flex
+        direction='column'
+        width='395px'
+        height='422px'
+        alignItems='center'
+        justifyContent='space-between'
+      >
+        <DefiModalHeader
```

Comment:
I wanted to also add a `<DefiModalBody />`, for the text nodes but I'll abstract it away in the `About Staking Rewards` work. I'd like to make sure to abstract the body nodes the right way so that the component is both reusable and styles match different copies/number of text nodes etc.